docs/developer/protocol/coolit.md

```diff
@@ -0,0 +1,84 @@
+# Coolit Protocol
+
+## Compatible devices
+
+| Device Name | USB ID | LED channels | Fan channels | Temperature channels | AIO |
+|:-----------:|:------:|:------------:|:------------:|:--------------------:|:---:|
+| Corsair Hydro H110i GT | `1b1c:0c04` | 0 | 2 + pump | 1 (internal) | Yes |
+
+This device also features a pump channel, controlled with the appropriate command (see below).
+
+## Command format
+
+Unless stated otherwise all multi-byte numbers used little endian.
+
+The device expect packets in the following order/format:
+
+1. Request ID byte (usually a counter that rolls over)
+2. Opcode byte
+3. Command byte
+4. Optional arguments
+
+See the `_build_data_package` function for data package construction.
+
```

Comment:
Can you add a quick description on the incoming/response packets?

It might also be good to mention that some commands/opcodes can be combined in a single packet (e.g. in `get_status`), and how that affects the response packet as well.

Reply:
Not going to lie, I'm quite unsure as to how the exact reply works. I mostly adapted the existing driver from previous PR...

Added mention for multiple data packages though.

Reply:
That's ok.